manifest/dev.baru/HeadCullingMask/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "HeadCullingMask",
```

Comment:
Are there still issues with `RenderTransformOveride`? I think some fixes were put towards that in January.

Reply:
yes

Reply:
Didn't find an issue for that specifically, may want to make an issue for it. https://github.com/Yellow-Dog-Man/Resonite-Issues/